foundation/foundation-archetype/test-project-archetype.sh

```diff
@@ -175,7 +175,7 @@ cp ../../../src/test/resources/test-project-archetype-script/maven-build-cache-c
 echo -e "\n# maven-suppress-warnings" >> Tiltfile
 
 echo -e "\nINFO: Running full build to generate project structure\n"
-mvn clean install || { echo -e '\n\n\t**** MAVEN BUILD FAILED ****\n\n' ; exit 1; }
+mvn clean install -Dmaven.build.cache.enabled=false || { echo -e '\n\n\t**** MAVEN BUILD FAILED ****\n\n' ; exit 1; }
```

Comment:
I: There are several other `mvn clean install` calls throughout the test.  I reasoned that only the first and last mattered, but that might be faulty logic. Happy to extend this to every `mvn clean install` call if that makes sense.

Reply:
My thought it why not extend it to every call. Could turn out later we were missing some bug in those middle ones that was getting cached. Not sure I fully understand the innerworkings of the cache, but no need to risk it.